Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -65,7 +65,8 @@
            11834.13,
            11723.17,
            12434.0,12634.0,12634.99,
-           12834.0,13034.0,13034.99]
+           12834.0,13034.0,13034.99,
+           12034.91]
```

Comment:
please try to keep it numerically ordered; also, 120XX is a wrong workflow to try; 116XX is more appropriate